server/src/services/media.service.ts

```diff
@@ -39,7 +39,8 @@ export class MediaService extends BaseService {
   private maliOpenCL?: boolean;
   private devices?: string[];
 
-  async handleQueueGenerateThumbnails({ force }: IBaseJob): Promise<JobStatus> {
+  @OnJob({ name: JobName.QUEUE_GENERATE_THUMBNAILS, queue: QueueName.THUMBNAIL_GENERATION })
```

Comment:
What happens if I put this decorator on two different methods in this Service?

Reply:
It throws an error and fails to start